moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -48,7 +48,7 @@ JogCalcs::JogCalcs(const JogArmParameters& parameters, const robot_model_loader:
   : parameters_(parameters), default_sleep_rate_(1000)
 {
   // Publish collision status
-  warning_pub_ = nh_.advertise<std_msgs::Bool>(parameters_.warning_topic, 1);
+  warning_pub_ = nh_.advertise<std_msgs::String>(parameters_.status_topic, 1);
```

Comment:
I thought the intent was to replace this with a new message type that included both a int32 (enum) and a string.  That way software that interacts could recover from errors returned this way and also print the warning string for humans.

Reply:
@tylerjw  Summarizing what we talked about yesterday: ideally there would be a ROS message type with an `int` field and a `string` field so I could publish both without making a separate PR to moveit_msgs. But there isn't, so... the new commits publish an int8 status code and provide a C++ interface for checking the status. A human-readable warning message is logged, too.